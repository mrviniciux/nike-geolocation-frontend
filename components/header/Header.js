import Head from 'next/head';
import Navbar from '../navbar/Navbar';

const Header = () => (
  <>
    <Head>
      <title>Grupo SBF</title>
      <meta
        name="description"
        content="Nike, Centauro, Fisia, NWB, Onefan, X3M, Fit Dance"
      />
      <link rel="icon" href="/favicon.svg" />
    </Head>

    <Navbar />
  </>
);

export default Header;
