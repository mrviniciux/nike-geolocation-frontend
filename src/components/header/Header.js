import Head from 'next/head';
import Navbar from '../navbar/Navbar';
import Pitchbar from './Pitchbar';
import Subheader from './Subheader';

const Header = () => (
  <>
    <Head>
      <title>Nike Geolocation</title>
      <meta
        name="description"
        content="Nike, Centauro, Fisia, NWB, Onefan, X3M, Fit Dance"
      />
      <link rel="icon" href="/favicon.svg" />
    </Head>

    <Subheader />
    <Navbar />
    <Pitchbar />
  </>
);

export default Header;
