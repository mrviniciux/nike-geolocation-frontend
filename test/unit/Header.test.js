/**
 * @jest-environment jsdom
 */

import React from 'react';
import Header from '../../src/components/header/Header';
import { render } from '@testing-library/react';

describe('Header rendering tests/snapshot', () => {
  it('should match snapshot', () => {
    const result = render(<Header />);

    expect(result).toMatchSnapshot();
  });
});
