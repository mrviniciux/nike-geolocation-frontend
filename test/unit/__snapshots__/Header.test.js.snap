// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header rendering tests/snapshot should match snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <div>
          <img
            src="/assets/logo_Jordan.svg"
          />
          <img
            src="/assets/logo_snkrs.svg"
          />
        </div>
        <ul>
          <li>
            <a
              href="#"
            >
              Ajuda
            </a>
          </li>
          <li>
            |
          </li>
          <li>
            <a
              href="#"
            >
              Acompanhe seu pedido
            </a>
          </li>
          <li>
            |
          </li>
          <li>
            <a
              href="#"
            >
              Junte-se a nós
            </a>
          </li>
          <li>
            |
          </li>
          <li>
            <a
              href="#"
            >
              Entrar
            </a>
          </li>
        </ul>
      </div>
      <nav
        id="SearchNavbar"
      >
        <div>
          <a
            href="/"
          >
            <img
              alt="Nike logo"
              src="/assets/logo_nike.svg"
            />
          </a>
        </div>
        <ul>
          <li>
            <a
              href="#"
            >
              Lançamento
            </a>
          </li>
          <li>
            <a
              href="#"
            >
              Masculino
            </a>
          </li>
          <li>
            <a
              href="#"
            >
              Feminino
            </a>
          </li>
          <li>
            <a
              href="#"
            >
              Infantil
            </a>
          </li>
          <li>
            <a
              href="#"
            >
              Ofertas
            </a>
          </li>
          <li>
            <a
              href="#"
            >
              SNKRS
            </a>
          </li>
        </ul>
        <div>
          <div>
            <span
              class="ant-input-affix-wrapper ant-input-affix-wrapper-lg"
            >
              <span
                class="ant-input-prefix"
              >
                <span
                  aria-label="search"
                  class="anticon anticon-search"
                  role="img"
                >
                  <svg
                    aria-hidden="true"
                    data-icon="search"
                    fill="currentColor"
                    focusable="false"
                    height="1em"
                    viewBox="64 64 896 896"
                    width="1em"
                  >
                    <path
                      d="M909.6 854.5L649.9 594.8C690.2 542.7 712 479 712 412c0-80.2-31.3-155.4-87.9-212.1-56.6-56.7-132-87.9-212.1-87.9s-155.5 31.3-212.1 87.9C143.2 256.5 112 331.8 112 412c0 80.1 31.3 155.5 87.9 212.1C256.5 680.8 331.8 712 412 712c67 0 130.6-21.8 182.7-62l259.7 259.6a8.2 8.2 0 0011.6 0l43.6-43.5a8.2 8.2 0 000-11.6zM570.4 570.4C528 612.7 471.8 636 412 636s-116-23.3-158.4-65.6C211.3 528 188 471.8 188 412s23.3-116.1 65.6-158.4C296 211.3 352.2 188 412 188s116.1 23.2 158.4 65.6S636 352.2 636 412s-23.3 116.1-65.6 158.4z"
                    />
                  </svg>
                </span>
              </span>
              <input
                class="ant-input ant-input-lg"
                placeholder="Buscar"
                type="text"
                value=""
              />
            </span>
          </div>
          <img
            src="/assets/icon_sacola.svg"
          />
        </div>
        <div>
          <img
            src="/assets/icon_sacola.svg"
          />
          <span
            aria-label="menu"
            class="anticon anticon-menu"
            role="img"
          >
            <svg
              aria-hidden="true"
              data-icon="menu"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M904 160H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8zm0 624H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8zm0-312H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8z"
              />
            </svg>
          </span>
        </div>
      </nav>
      <div>
        <div>
          <img
            src="assets/icon_seta_esquerda.svg"
          />
        </div>
        <div>
          <img
            src="/assets/logo_nike_snkrs.svg"
          />
          <span>
            Fique por dentro dos lançamentos
             
            <b>
              <u>
                SNKRS
              </u>
            </b>
          </span>
        </div>
        <div>
          <img
            src="assets/icon_seta_direita.svg"
          />
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <div>
        <img
          src="/assets/logo_Jordan.svg"
        />
        <img
          src="/assets/logo_snkrs.svg"
        />
      </div>
      <ul>
        <li>
          <a
            href="#"
          >
            Ajuda
          </a>
        </li>
        <li>
          |
        </li>
        <li>
          <a
            href="#"
          >
            Acompanhe seu pedido
          </a>
        </li>
        <li>
          |
        </li>
        <li>
          <a
            href="#"
          >
            Junte-se a nós
          </a>
        </li>
        <li>
          |
        </li>
        <li>
          <a
            href="#"
          >
            Entrar
          </a>
        </li>
      </ul>
    </div>
    <nav
      id="SearchNavbar"
    >
      <div>
        <a
          href="/"
        >
          <img
            alt="Nike logo"
            src="/assets/logo_nike.svg"
          />
        </a>
      </div>
      <ul>
        <li>
          <a
            href="#"
          >
            Lançamento
          </a>
        </li>
        <li>
          <a
            href="#"
          >
            Masculino
          </a>
        </li>
        <li>
          <a
            href="#"
          >
            Feminino
          </a>
        </li>
        <li>
          <a
            href="#"
          >
            Infantil
          </a>
        </li>
        <li>
          <a
            href="#"
          >
            Ofertas
          </a>
        </li>
        <li>
          <a
            href="#"
          >
            SNKRS
          </a>
        </li>
      </ul>
      <div>
        <div>
          <span
            class="ant-input-affix-wrapper ant-input-affix-wrapper-lg"
          >
            <span
              class="ant-input-prefix"
            >
              <span
                aria-label="search"
                class="anticon anticon-search"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  data-icon="search"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M909.6 854.5L649.9 594.8C690.2 542.7 712 479 712 412c0-80.2-31.3-155.4-87.9-212.1-56.6-56.7-132-87.9-212.1-87.9s-155.5 31.3-212.1 87.9C143.2 256.5 112 331.8 112 412c0 80.1 31.3 155.5 87.9 212.1C256.5 680.8 331.8 712 412 712c67 0 130.6-21.8 182.7-62l259.7 259.6a8.2 8.2 0 0011.6 0l43.6-43.5a8.2 8.2 0 000-11.6zM570.4 570.4C528 612.7 471.8 636 412 636s-116-23.3-158.4-65.6C211.3 528 188 471.8 188 412s23.3-116.1 65.6-158.4C296 211.3 352.2 188 412 188s116.1 23.2 158.4 65.6S636 352.2 636 412s-23.3 116.1-65.6 158.4z"
                  />
                </svg>
              </span>
            </span>
            <input
              class="ant-input ant-input-lg"
              placeholder="Buscar"
              type="text"
              value=""
            />
          </span>
        </div>
        <img
          src="/assets/icon_sacola.svg"
        />
      </div>
      <div>
        <img
          src="/assets/icon_sacola.svg"
        />
        <span
          aria-label="menu"
          class="anticon anticon-menu"
          role="img"
        >
          <svg
            aria-hidden="true"
            data-icon="menu"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M904 160H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8zm0 624H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8zm0-312H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8z"
            />
          </svg>
        </span>
      </div>
    </nav>
    <div>
      <div>
        <img
          src="assets/icon_seta_esquerda.svg"
        />
      </div>
      <div>
        <img
          src="/assets/logo_nike_snkrs.svg"
        />
        <span>
          Fique por dentro dos lançamentos
           
          <b>
            <u>
              SNKRS
            </u>
          </b>
        </span>
      </div>
      <div>
        <img
          src="assets/icon_seta_direita.svg"
        />
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
